Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -48,7 +48,7 @@ module Homebrew
   module_function
 
   def bump_formula_pr
-    @bump_args = Homebrew::CLI::Parser.parse do
+    @args = Homebrew::CLI::Parser.parse do
```

Comment:
Thinking about this more: I think it would be ideal if you could always just use `Homebrew.args` (or `args` if inside the `Homebrew` module already) and have `Homebrew::CLI::Parser.parse` add to the existing `args` for global values rather than having two sets that you need to query at separate times. Thoughts?

Reply:
Seems sensible to store everything in `Homebrew.args` I'll get this to work and if there are no issues will update the PR.

Reply:
@MikeMcQuaid , the PR is updated now.